.github/ISSUE_TEMPLATE/client_enhancement.md

```diff
@@ -2,7 +2,7 @@
 name: Client Enhancement
 about: Request a new feature to be added to NATS clients
 labels: enhancement, client
-assignees: ripienaar, wallyqs, scottf, aricart, ColinSullivan1, kozlovic, derekcollison, matthiashanel, Jarema
```

Comment:
I'm fine being removed from the .NET client feature requests. But in terms of high level / overall feature requests it's a good idea to keep the NATS product manager involved.  :)

Reply:
fixed